examples/run/task-run-private.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: pipeline.knative.dev/v1alpha1
+kind: TaskRun
+metadata:
+  name: pull-private-image-run
+  labels:
+    expect: succeeded
```

Comment:
Looking at the `e2e` scripts, we validate only taskrun and pipelinerun that have those labels, am I right ? (cc @shashwathi @ImJasonH)

Reply:
Asking this because there is existing yaml test that fail
```
I0123 14:05:57.625] default                  pod/build-push-run-pod-0fa98d                                        0/1       Init:Error   0          58s
```

Reply:
I don't think so. `expect` labels were supported in Build  but not in taskrun. yaml e2e test checks the status to be succeeded.

Reply:
@shashwathi :thinking: but we do filter them based on an `expect` lable [here](https://github.com/knative/build-pipeline/blob/master/test/e2e-common.sh#L65), don't we ?

Reply:
then test script is wrong(I think). There is no label as far as I know. Test script should check status to be succeeded.